libminifi/test/libtest/CMakeLists.txt

```diff
@@ -16,5 +16,11 @@
 # specific language governing permissions and limitations
 # under the License.
 #
+if (NOT ENABLE_ALL AND NOT ENABLE_PROMETHEUS AND NOT ENABLE_GRAFANA_LOKI AND NOT ENABLE_CIVET)
+    message("CivetWeb was disabled for MiNiFi, but will be enabled for tests")
+    include(GetCivetWeb)
+    get_civetweb()
+endif()
+
```

Comment:
I think this should just always run and make sure that civet is available when tests need it. I'd remove the `if` condition and make sure that including GetCivetWeb from multiple files only created the target once.

Reply:
Good idea, updated in https://github.com/apache/nifi-minifi-cpp/pull/1929/commits/6c1f137170ca6a00a79b19e3427f9144e1bcefa7